opencti-platform/opencti-graphql/src/database/smtp.js

```diff
@@ -27,6 +31,18 @@ if (conf.get('smtp:username') && conf.get('smtp:username').length > 0) {
 
 export const transporter = nodemailer.createTransport(smtpOptions);
 
+export const smtpConfiguredEmail = (settings) => {
+  return ALLOW_EMAIL_REWRITE ? settings.platform_email : conf.get('smtp:username');
```

Comment:
SMTP username is not always an email address (eg SMTP server connected on LDAP)
We should add a new conf to specify the 'forced' email address, that could replace the allow_email_rewrite.
eg smtp.enforced_from_email : if set we use it else the setting 

->
```
export const smtpConfiguredEmail = (settings) =>conf.get('smtp.enforced_from_email') ?? settings.platform_email;
```

Reply:
approach changed / issue rewritten